src/demo/styles.css

```diff
@@ -49,6 +49,9 @@ ul, li {
   padding-left: 0;
   font-family: Arial, Helvetica, sans-serif;
 }
+pre {
```

Comment:
It feels like a bad idea to globally override a block level element to display inline. I don't see this used anywhere? Can you just remove it? If not, it should be scoped with the component it's used in.